src/controllers.php

```diff
@@ -18,9 +19,27 @@
 })
 ->bind('login');
 
+$app->post('/login', function (Request $request) use ($app) {
+    $name = $request->get('name');
+    $password = $request->get('password');
+    $user = $app['db']->fetchAssoc('SELECT * FROM users WHERE name = ?', [$name]);
+
+    if (password_verify($password, $user['password'])) {
+        if ($app['session']->get('is_logged') != 1) {
+            $app['session']->set('is_logged', 1);
+            $app['session']->set('user', ['name' => $name]);
+            $app['session']->start();
```

Comment:
to chyba  powinno byc na poczatku :)

Reply:
wogle to jest potrzebne? wedlug dokumentacji chyba nie

Reply:
Bez 
$app['session']->set('user', ['name' => $name]);
$app['session']->start();
spokojnie oczywiście też działa jak powinno. Czy taka zmiana będzie wystarczająca? Czy masz na myśli coś więcej?

Reply:
poprostu bym start usunal bo to powinno sie odpalac wewnatrz frameworka same

Reply:
zrobione

Reply:
zrobione bis